cdap-docs/_common/_source/index.rst

```diff
@@ -93,7 +98,8 @@ on the installation, monitoring and diagnosing fully distributed CDAP in a Hadoo
     guarantees, transaction service maintenance, troubleshooting and introduces the CDAP UI
   - **Troubleshooting and Appendices:** Installation and configuration troubleshooting
     tips, and appendices that cover the CDAP installation and security configuration files
-
+  - **Modes:** CDAP can be run in-memory for unit testing, standalone for testing on a developer's 
```

Comment:
Good idea. I have edited down line ~101, and moved the "standalone for development and testing" up to line 40-42. Please re-review...